### PR TITLE
LNorris DCS-707 amended date format for the date input on /bookings p…

### PR DESCRIPTION
**Pull request by louisenorris**
…age to match the h1 heading format

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
     const minDate = $(element).data('min-date') ? $(element).data('min-date') : disablePastDates ? '0' : undefined
-    const dateFormat = 'dd/mm/yy'
+    const dateFormat = $(element).attr('date-format') || 'dd/mm/yy'
 
     $(element).datepicker({
       dateFormat,
-      showOtherMonths: true,
-      selectOtherMonths: true,
+      showOtherMonths: false,
+      selectOtherMonths: false,
```
Comment (andrewrlee) [inline static/js/main.js:12]: not sure we wanted these changes, wasn't this just for testing?

File: static/js/main.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
       maxDate: maxDate,
       minDate: minDate,
     })
   })
 
-  $('.js-prisoner-search-clear-alerts').click(function(e) {
+  $('.js-prisoner-search-clear-alerts').click(function (e) {
```
Comment (andrewrlee) [inline static/js/main.js:18]: meh.. unnecessary whitespace

File: sass/_horizontal-content.scss [hunk 3]
```diff
@@ -43,7 +43,7 @@
     }
 
     .date-picker-container input {
-      width: 130px !important;
+      width: 185px !important;
```
Comment (andrewrlee) [inline sass/_horizontal-content.scss:46]: does this affect both date inputs? if so does it matter?

File: static/js/main.js (hunk 1 above)
Comment (louisenorris) [reply static/js/main.js:12]: Ah - apologies! I'll change that asap..

File: sass/_horizontal-content.scss (hunk 3 above)
Comment (louisenorris) [reply sass/_horizontal-content.scss:46]: It only affects the date input on the video link bookings list page at the moment as it's got the horizontal-form classname and that page is currently the only page with horizontal form layout. So all good for now but may matter with future development... tbc.

**Review by andrewrlee — APPROVED** [review]
LGTM
